src/shapes.lisp

```diff
@@ -10,8 +10,9 @@
 
 (defun point (x y)
   (declare (type real x y))
-  (with-pen (make-pen :fill (pen-stroke (env-pen *env*)))
-    (rect x y 1 1)))
+  (let ((weight (or (pen-weight (env-pen *env*)) 1)))
+    (with-pen (make-pen :fill (pen-stroke (env-pen *env*)))
+      (circle x y weight))))
```

Comment:
I've just realized that this changes how points are rendered. they are twice the size (easily fixed with `(/ weight 2)`, but also translated 

![diff](https://github.com/vydd/sketch/assets/2054352/9ecb1a40-1799-4721-b87a-6608255f077c)

Reply:
(here's the code I used)

```lisp
(defsketch point ()
  (background +white+)
  (scale 40)
  (with-pen (make-pen :fill (rgb 0 1 0 0.5))
    (rect 5 5 1 1)
    (rect 4 6 1 1)
    (rect 4 7 1 1)
    (rect 5 7 1 1)
    (rect 6 7 1 1))
  (with-pen (make-pen :stroke (rgb 1 0 0 0.2))
    (point 5 5)
    (point 4 6)
    (point 4 7)
    (point 5 7)
    (point 6 7)))
```

Reply:
I would argue that circles are more correct here, as their center is in the desired location while with rectangles they are sifted to the right & bottom (or top when `y-axis` is `:up`)

Reply:
I checked how [Processing does it](https://processing.org/reference/point_.html), and it is indeed centering as the two of you are suggesting. Point was supposed to copy Processing's behavior, so let's say this is fixing the bug :) (I kinda see both as valid choices).

width still needs to be fixed, though

Reply:
Completed: e34bcd2b629e1c504ae2e984302b545dc9f78432.